stable/prometheus-operator/hack/sync_prometheus_rules.py

```diff
@@ -63,6 +63,7 @@ def new_representer(dumper, data):
     'PrometheusOperatorDown': '.Values.prometheusOperator.enabled',
     'NodeExporterDown': '.Values.nodeExporter.enabled',
     'CoreDNSDown': '.Values.kubeDns.enabled',
+    'AlertmanagerDown': '.Values.alertManager.enabled'
```

Comment:
Please add comma at the end

Reply:
And in addition to changing the script, please actually run it and commit changes which it will introduce.

Reply:
Added the comma and ran the script to re-sync the rules.

Reply:
Also cleaned up description slightly to adjust to changes.